holo-ospf/src/northbound/state.rs

```diff
@@ -2845,7 +2848,8 @@ fn load_callbacks_ospfv3() -> Callbacks<Instance<Ospfv3>> {
         .path(ospf::areas::area::database::area_scope_lsa_type::area_scope_lsas::area_scope_lsa::ospfv3::body::e_intra_area_prefix::e_intra_prefix_tlvs::intra_prefix_tlv::sub_tlvs::PATH)
         .get_iterate(|_instance, args| {
             let prefix = args.parent_list_entry.as_ospfv3_intra_area_lsa_prefix().unwrap();
-            let iter = prefix.unknown_stlvs.iter().map(ListEntry::UnknownTlv).chain(std::iter::once(ListEntry::Ospfv3PrefixSids(&prefix.prefix_sids)));
+            let iter = prefix.unknown_stlvs.iter().map(ListEntry::UnknownTlv)
+                        .chain(std::iter::once(ListEntry::Ospfv3PrefixSids(&prefix.prefix_sids))).chain(std::iter::once(ListEntry::Ospfv3Biers(&prefix.bier)));
```

Comment:
This chain of chain seems to cause an empty dict for `PrefixSids`

```json
"sub-tlvs": [
  {},
  {
    "holo-ospf:bier-info-sub-tlvs": {
      "bier-info-sub-tlv": [
        {
          "sub-domain-id": 0,
          "mt-id": 0,
          "bfr-id": 6,
          "bar": 0,
          "ipa": 0
        }
      ]
    }
  }
]
```

Reply:
Indeed. The container within the Sub-TLVs list makes things more difficult. This should do the trick:
```rs
            let mut iter: Box<dyn Iterator<Item = ListEntry<'_, _>>>;
            iter = Box::new(prefix.unknown_stlvs.iter().map(ListEntry::UnknownTlv));
            if !prefix.prefix_sids.is_empty() {
                iter = Box::new(iter.chain(std::iter::once(ListEntry::Ospfv3PrefixSids(&prefix.prefix_sids))));
            }
            if !prefix.bier.is_empty() {
                iter = Box::new(iter.chain(std::iter::once(ListEntry::Ospfv3Biers(&prefix.bier))));
            }
            Some(iter)
```
In the long term, we need a major refactoring of the northbound layer to simplify the process of writing these callbacks.

Reply:
That indeed fixed the issue, thanks! Patched in d5d2e093c9bae984d1075ba5c809870292eb78a1